main.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask
+import random
+
+app = Flask(__name__)
+random_number = random.randint(0, 9)
+
+
+@app.route('/')
+def home():
+    return '<h1>Guess a number between 0 and 9!</h1>' \
+           '<h3>Type a "/" after the address in your search bar, followed by your guess.</h3>' \
+           '<img src="https://media.giphy.com/media/3o7aCSPqXE5C6T8tBC/giphy.gif">'
+
+
+@app.route("/<int:guess>")
+def guess_number(guess):
+    if guess > random_number:
```

Comment:
Looks good.  Some of this is duplicated so I'd probably create a dictionary with values for: `message`, `color`, and `image`, stash it at the top, and then grab those values during the game.

Reply:
I included a json file to hold the data for the styling, img elements, and message outputs